internal/metrics/metricsManager.go

```diff
@@ -108,6 +114,8 @@ type API interface {
 	FileSystemUsage(usageInPercentage float64)
 	MonitoredHostsCount(monitoredHosts int64)
 	MonitoredClusterCount(monitoredClusters int64)
+	NetworkLatencyBetweenHosts(clusterVersion string, emailDomain string, sourceRole, targetRole models.HostRole, latency float64)
+	PacketLossBetweenHosts(clusterVersion string, emailDomain string, sourceRole, targetRole models.HostRole, packetLoss float64)
```

Comment:
No need for the emailDomain.
Is there a readon to add the source and target roles?

Reply:
The labels were discussed with @gamli75 and @avishayt to use these labels in the metric. The reason for adding the roles is to provide a better visibility between hosts with the same role and different role. Currently we are only validating against hosts with the same role, but knowing the value between master<->worker can also help us get a better picture of the average cluster latency/packet loss overall.

Reply:
roles is OK, but email domain should definitely not be in the metric
@avishayt , @gamli75

Reply:
About the roles, we need to measure the latency between all the hosts, regardless of the roles.
Validation is something else.
The metrics should be sent on the values we measured and not on the result of the validations

Reply:
As discussed, I'll remove the `emailDomain` from both metrics.